UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminOverlayPanel.cs

```diff
@@ -89,11 +89,11 @@ void FollowTarget()
 			// check container:
 			if(targetObjBehaviour != null)
 			{
-				if (targetObjBehaviour.ContainedInContainer != null)
+				if (targetObjBehaviour.ContainedInObjectContainer != null)
 				{
-					if (targetObjBehaviour.ContainedInContainer.transform != target)
+					if (targetObjBehaviour.ContainedInObjectContainer.transform != target)
```

Comment:
Unnecessary nesting is going on here

Reply:
it's going to be really massively long if statement if I do combine them all

Reply:
you can just put it in functions?

Reply:
But isn't that even more messy?

Reply:
No?